### PR TITLE
Update some keymaps

### PR DESCRIPTION
**Pull request by fauxpark**

### CODE REVIEW

File: public/keymaps/h/handwired_swiftrax_cowfish_default.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
```
Comment (noroadsleft) [inline public/keymaps/h/handwired_swiftrax_cowfish_default.json:1]: This keymap won't work unless and until https://github.com/qmk/qmk_firmware/pull/18984 is back-ported to `qmk:master`.

File: public/keymaps/w/wuque_promise87_ansi_default.json [hunk 2]
```diff
@@ -5,20 +5,20 @@
...
-      "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC", "KC_BSPC",    "KC_INS",  "KC_HOME", "KC_PGUP",
-      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",               "KC_DEL",  "KC_END",  "KC_PGDN",
+      "KC_ESC",  "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",  "KC_F13",               "KC_PSCR", "KC_SCRL", "KC_PAUS",
+      "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC", "KC_BSPC",   "KC_INS",  "KC_HOME", "KC_PGUP",
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",              "KC_DEL",  "KC_END",  "KC_PGDN",
```
Comment (fauxpark) [inline public/keymaps/w/wuque_promise87_ansi_default.json:10]:
```suggestion
      "KC_ESC",  "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",  "KC_F13",                "KC_PSCR", "KC_SCRL", "KC_PAUS",
      "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC", "KC_BSPC",    "KC_INS",  "KC_HOME", "KC_PGUP",
      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",               "KC_DEL",  "KC_END",  "KC_PGDN",
```

File: public/keymaps/w/wuque_promise87_ansi_default.json [hunk 3]
```diff
@@ -5,20 +5,20 @@
...
       "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",
-      "KC_LSFT", "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "KC_RSFT",                          "KC_UP",
-      "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",                                   "KC_RALT", "MO(1)",   "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT"
+      "KC_LSFT", "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "KC_RSFT",                         "KC_UP",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",                                   "KC_RALT", "MO(1)",   "KC_RCTL",              "KC_LEFT", "KC_DOWN", "KC_RGHT"
```
Comment (fauxpark) [inline public/keymaps/w/wuque_promise87_ansi_default.json:13]:
```suggestion
      "KC_LSFT", "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "KC_RSFT",                          "KC_UP",
      "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",                                   "KC_RALT", "MO(1)",   "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT"
```

File: public/keymaps/w/wuque_promise87_ansi_default.json [hunk 4]
```diff
@@ -5,20 +5,20 @@
...
-      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",    "_______", "_______", "QK_BOOT",
-      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",               "_______", "_______", "_______",
+      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",              "_______", "_______", "_______",
+      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",   "_______", "_______", "QK_BOOT",
+      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",              "_______", "_______", "_______",
```
Comment (fauxpark) [inline public/keymaps/w/wuque_promise87_ansi_default.json:18]:
```suggestion
      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",               "_______", "_______", "_______",
      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",    "_______", "_______", "QK_BOOT",
      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",               "_______", "_______", "_______",
```

File: public/keymaps/w/wuque_promise87_ansi_default.json [hunk 5]
```diff
@@ -5,20 +5,20 @@
...
       "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",            "_______",
-      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",                          "_______",
-      "_______", "_______", "_______",                       "_______", "_______", "_______",                                  "_______", "_______", "_______",               "_______", "_______", "_______"
+      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",                         "_______",
+      "_______", "_______", "_______",                       "_______", "_______", "_______",                                  "_______", "_______", "_______",              "_______", "_______", "_______"
```
Comment (fauxpark) [inline public/keymaps/w/wuque_promise87_ansi_default.json:21]:
```suggestion
      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",                          "_______",
      "_______", "_______", "_______",                       "_______", "_______", "_______",                                  "_______", "_______", "_______",               "_______", "_______", "_______"
```

**Comment by fauxpark** [comment]
Bump

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**